src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -119,7 +119,7 @@
         <{{ element|default('div') }} class="custom-file">
             {%- set type = type|default('file') -%}
             {{- block('form_widget_simple') -}}
-            <label for="{{ form.vars.id }}" class="custom-file-label">Choose File</label>
+            <label for="{{ form.vars.id }}" class="custom-file-label">{{ attr.placeholder | default("") }}</label>
```

Comment:
`attr.placeholder|default('')` for consistency

Reply:
also it should be translatable i guess :thinking:

Reply:
I fixed the inconsistency (my tunnel vision was strong there I guess).